app/src/main/java/org/mozilla/focus/utils/HomeTileUtils.java

```diff
@@ -0,0 +1,28 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils;
+
+import android.content.Context;
+import org.json.JSONObject;
+import java.io.InputStream;
+
+public class HomeTileUtils {
+    public static JSONObject loadDefaultTiles(Context context) {
+        String json;
+        try {
+            InputStream is = context.getAssets().open("defaults/DefaultTiles.json");
```

Comment:
If you do this in kotlin, you can make this read simpler ([according to SO](https://stackoverflow.com/a/39500046)):
```kotlin
val inputAsString = inputStream.bufferedReader().use { it.readText() }  // defaults to UTF-8
```

tbh, I think we should avoid adding any more Java files because it makes interop more difficult.

Reply:
And that means no need to handle the `try/catch/finally` cases